docs/docs/server-administration/dns.md

```diff
@@ -49,24 +62,22 @@ This way you can set up Master -> Slave or Master <-> Master <-> Master cluster.
 
 There is no limitation on how to chain DNS servers.
 
-### DNS Cluster with the Hestia API (Master -> Slave)
+### Master -> Slave DNS cluster with the Hestia API
 
-1. Create a new user on the Hestia server that will act as a “Slave”. Make sure it uses the username of "dns-user" or has the role `dns-cluster`
-2. In `/usr/local/hestia/conf/hestia.conf`, change `DNS_CLUSTER_SYSTEM='hestia'` to `DNS_CLUSTER_SYSTEM='hestia-zone'`.
-3. On the master server, open `/etc/bind/named.conf.options`, do the following changes, then restart bind9 with `systemctl restart bind9`.
+Preparing your **Slave** server(s):
 
-   ```bash
-   # Change this line
-   allow-transfer { "none"; };
-   # To this
-   allow-transfer { your.slave.ip.address; };
-   # Or this, if adding multiple slaves
-   allow-transfer { first.slave.ip.address; second.slave.ip.address; };
-   # Add this line, if adding multiple slaves
-   also-notify { second.slave.ip.address; };
-   ```
+1. Whitelist your master server IP in **Configure Server** -> **Security** -> **Allowed IP addresses for API**
+2. Enable API access for admins (or all users).
+3. Create an API key under the **admin** user with at least the **sync-dns-cluster** permission.
+4. Create a new DNS sync user as follows:
 
-4. On the slave server, open `/etc/bind/named.conf.options`, do the following changes, then restart bind9 with `systemctl restart bind9`:
+- Username of "dns-user"
+- Has email address (something generic)
+- Has the role `dns-cluster`
+- Set 'Do not allow user to log in to Control Panel'
```

Comment:
if you indent this list, it should fix the issue right under here. it should be indented anyways, as it's part of the 4th item in the list.